frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketItem/ticketItemBaseInfo/ticketItemBaseInfo.component.tsx

```diff
@@ -16,40 +16,33 @@
  */
```

Comment:
When the author name is too long gets truncated, but it seems it is not responsive to the actual width
![image](https://github.com/3drepo/3drepo.io/assets/43547703/02a19f5a-0d7a-484f-abac-deca9cabd980)